### DISCUSSION TITLE
How to open the data in qgis

### DISCUSSION THREAD

**Discussion by justinelliotmeyers** [OvertureMaps/data#4] [Discussion]
How can this data be opened in qgis? Drag and drop doesn't seem to work.

Thanks

**Comment by jenningsanderson** [comment]
Drag-n-drop in QGIS works with GDAL > 3.5; see examples here: https://docs.overturemaps.org/examples/QGIS/

**Reply by justinelliotmeyers** [reply]
Thank you!
